apps/www/components/Climatelab/Counter.js

```diff
@@ -36,8 +75,16 @@ const query = gql`
   }
 `
 
-const Counter = ({ data }) => {
+const Counter = ({
+  data,
+  smallNumbers,
```

Comment:
I am very confused `smallNumbers` to be a weird name. Small in relation to what? Is there a bigNumbers hiding somewhere else? 
(After looking at publikator...)
Suggestion: How about `statisticsBreakout`, or `demographics` or something like that? And `showStatistics` or `showDemographics` instead of `extended`?

Reply:
sounds good, will do the changes